delete_me.py

```diff
@@ -0,0 +1,3 @@
+good_var_name = 1
```

Comment:
The file has increased in 3 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._